relay-server/src/lib.rs

```diff
@@ -310,3 +310,20 @@ pub fn run(config: Config) -> anyhow::Result<()> {
     relay_log::info!("relay shutdown complete");
     Ok(())
 }
+
+#[cfg(test)]
+mod tests {
+    use relay_base_schema::events::EventType;
+    use relay_event_schema::protocol::{Event, EventId, LenientString};
+    use relay_protocol::Annotated;
+
+    pub fn mocked_event(event_type: EventType, transaction: &str, release: &str) -> Event {
```

Comment:
Same function was used multiple test modules so i did this to reduce duplication

Reply:
By convention, `mod tests` is used exclusively for unit tests of the containing module. If you really want to deduplicate this helper, consider to place it in a module under `utils`. However, in this case it's probably not worth it. Tests should be as self-contained and explicit as possible, so it's better not to import such functions from other modules.

I would therefore suggest to revert this.

Reply:
alright, that makes sense, I've reverted it!